src/Storages/MergeTree/IMergeTreeDataPart.cpp

```diff
@@ -1334,7 +1337,7 @@ void IMergeTreeDataPart::loadColumns(bool require)
 
         /// If there is no file with a list of columns, write it down.
         for (const NameAndTypePair & column : metadata_snapshot->getColumns().getAllPhysical())
-            if (getDataPartStorage().exists(getFileNameForColumn(column) + ".bin"))
+            if (getFileNameForColumn(column))
```

Comment:
This will not work since `getFileNameForColumn` requires `serilizations`, which will be filled only at the end of `loadColumns`, should be fixed by - https://github.com/ClickHouse/ClickHouse/pull/64757